scattering/scattering.py

```diff
@@ -100,6 +100,91 @@ def structure_factor(trj, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False)
         S[i] = num/denom
     return Q, S
 
+
+def structure_factor_pair(trj, pair, Q_range=(0.5, 50), n_points=1000, framewise_rdf=False, method='fz'):
+    rho = np.mean(trj.n_atoms / trj.unitcell_volumes)
+    L = np.min(trj.unitcell_lengths)
+
+    top = trj.topology
+    
+    elements_i = set()
+    elements_j = set()
+    for a in top.atoms:
+        string = a.element.symbol
+        if string[-2:] == pair[0]:
```

Comment:
I'm assuming the `pair` argument is a list.  Could you clarify the purpose of `string[-2:]`.  What this is doing is grabbing a chunk of `string` starting with the second to last index and ending at the last index.  

If `pair` for example is `[ca, an]` for a cation-anion pair, then I don't think these conditionals will ever be met because there isn't an `element.symbol` that equals `ca` or `an`.

Reply:
because I renamed the atoms in gro file as shown in the picture. 
<img width="204" alt="Screen Shot 2020-07-08 at 12 59 24 PM" src="https://user-images.githubusercontent.com/57978910/86954085-03c8f200-c11b-11ea-9822-fdbd7f63f7d2.png">

The printed result is this picture below.
<img width="453" alt="Screen Shot 2020-07-08 at 1 01 45 PM" src="https://user-images.githubusercontent.com/57978910/86954216-34109080-c11b-11ea-8a06-bf62310f89f5.png">